ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -145,6 +193,11 @@ export default React.memo<Props>(function VideoJs(props: Props) {
 
       player = videojs(el, videoJsOptions, () => {
         if (player) {
+          player.one('play', onInitialPlay);
```

Comment:
I think this needs to be `on`

Reply:
We only need to run this one time, so `one` is used.  If `on` is used, the button will re-appear every time "Play" is pressed while sound is muted.

Reply:
Interesting. I didn't know videojs supported that.

I tried several videos that started muted and didn't see the button. Can you double-check that it's working?

Reply:
Yeah, I just rebased to confirm the `one` before a few minutes ago and the button is working.

Reply:
Tried a fresh clone on another linux VM and it's ok too.  Let me try on Windows.

Reply:
Ok, if I clear my Electron settings and never touch the videojs player buttons, then I can replicate the issue.  Once I toggled the mute button, it will work from then on.  See if that works for you to test the other aspects.

This is again due to #4358 not reflecting the actual state of the audio.  Let me dig further.  
By the way, in `handleKeyDown` of `VideoJs`, do you know why the `HTMLVideoElement` was used when the `Player` is available? This is my wild guess on what's causing the mismatch.

Reply:
I think I've figured it out.  The Mute and Volume are independent controls and we need to handle two types of "mute":
- Volume icon is pressed to mute
- Volume slider moved to 0%
Fix commited.